core/src/main/scala/sttp/tapir/Schema.scala

```diff
@@ -301,6 +301,8 @@ object Schema extends LowPrioritySchema with SchemaCompanionMacros {
   implicit def schemaForOption[T: Schema]: Schema[Option[T]] = implicitly[Schema[T]].asOption
   implicit def schemaForArray[T: Schema]: Schema[Array[T]] = implicitly[Schema[T]].asArray
   implicit def schemaForIterable[T: Schema, C[X] <: Iterable[X]]: Schema[C[T]] = implicitly[Schema[T]].asIterable[C]
+  implicit def schemaForSet[T: Schema, C[X] <: scala.collection.Set[X]]: Schema[C[T]] = 
```

Comment:
This relies on Scala implicit priority rules, specifically the fact that `schemaForSet` is "more specific" than `schemaForIterable`. It seems to work fine in tests, but I always feel a bit anxious when I need to rely on these rules, and in this case it seems to be the first implicit definition to do so.
I just hope we don't introduce some crazy implicit ambiguities into tapir user's projects...

Reply:
we'll find out soon enough :)